autopush/settings.py

```diff
@@ -51,6 +51,7 @@ def __init__(self,
                  pingConf=None,
                  resolve_hostname=False,
                  min_ping_interval=20,
+                 pong_delay=0,
                  max_data=4096,
```

Comment:
nit, we should make max_data a config option as well. It's a number and we may want to change it.